src/charm.py

```diff
@@ -526,6 +526,18 @@ def _restart(self, _) -> None:
         """
         logger.debug("Restarting mysqld daemon")
         if service_restart(SERVICE_NAME):
+
+            # when restart done right after cluster creation (e.g bundles)
+            # or for single unit deployments, it's necessary reboot the
+            # cluster from outage to restore unit as primary
+            if self.app_peer_data["units-added-to-cluster"] == "1":
```

Comment:
seems like you have agressive wrapping here, canonical standards are 99 chars

Reply:
this was actually manually wrapped. It bothers me when the comment stretches far right from the thing it's about - ocd vibes at play here